### PR TITLE
Add functions to calculate properties from entropy and pressure.

### PR DESCRIPTION
**Pull request by eslickj**
See title.  Also adds better handling of the triple point for T(h, P), T(s, P), T_sat(P), and P_sat(T).  I know this is a little tough to review, but there are tests in the other part of this that goes into idaes-dev.

### CODE REVIEW

**Comment by eslickj** [comment]
Missed mentioning one other change.  If the Temperature goes too high or too low, I used to move it to something that would sort of work, and return a result.  Now, instead, I just return an error.  With Ipopt, this will just cause the step to cut back.  You'll also be able to do halt on ampl error and track down where it's happening.

File: src/helmholtz/iapws95/iapws95_param.h [hunk 1]
```diff
@@ -53,6 +53,8 @@ const s_real R = 0.46151805;  // Specific gas constant (kJ/kg/K)
 const s_real T_c = 647.096;   // Critical T (K)
 const s_real rho_c = 322;     // Critical density (kg/m^3)
 const s_real P_c = 2.2064e4;  // Critical Pressure (kPa)
+const s_real T_t = 273.16;    // Tripple point temperature (K)
```
Comment (andrewlee94) [inline src/helmholtz/iapws95/iapws95_param.h]: Typo: Triple, also below.

File: src/helmholtz/swco2/swco2_param.h [hunk 2]
```diff
@@ -47,6 +47,8 @@ const s_real R = 0.1889241;  // Specific gas constant (kJ/kg/K)
 const s_real T_c = 304.128;   // Critical T (K)
 const s_real rho_c = 467.6;   // Critical density (kg/m^3)
 const s_real P_c = 7377.3;    // Critical Pressure (kPa)
+const s_real T_t = 216.592;   // Tripple point temperature (K)
```
Comment (andrewlee94) [inline src/helmholtz/swco2/swco2_param.h]: Typo: Triple, and below.

**Comment by eslickj** [comment]
@andrewlee94, yeah we're working on it.  @dangunter started putting together docker containers we should be able to automatically build and test these.  I hope to have that going soon.  I'm not really sure how we're going to manage this but we'll figure it out.  I think we're going to start building for more Linux versions too.

**Comment by eslickj** [comment]
Okay, I fixed the typo.  We are slowly move toward automatic building and testing of these, so hopefully reviewing this won't require too much faith in my testing.

**Pull request merged**